src/components/social-media/SocialIconContainer.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC, useEffect, useState } from 'react'
+
+import facebookSrc from '../../images/social-icons/facebook.svg'
+import twitterSrc from '../../images/social-icons/twitter.svg'
+import linkedInSrc from '../../images/social-icons/linked-in.svg'
+import instagramSrc from '../../images/social-icons/instagram.svg'
+import SocialMediaLink from './SocialMediaLink'
+import { getThemeLargeScreenWidth } from 'utils/site-theme'
+
+const socialMediaDetails = [
+  {
+    href: 'https://www.instagram.com/distributeaid',
+    src: instagramSrc,
+    altText: 'Instagram icon',
+  },
+  {
+    href: 'https://twitter.com/DistributeAid',
+    src: twitterSrc,
+    altText: 'Twitter icon',
+  },
+  {
+    href: 'https://www.linkedin.com/company/distribute-aid',
+    src: linkedInSrc,
+    altText: 'LinkedIn icon',
+  },
+  {
+    href: 'https://www.facebook.com/DistributeAidDotOrg/',
+    src: facebookSrc,
+    altText: 'Facebook icon',
+  },
+]
+
+interface Props {
+  position: 'footer' | 'side'
+}
+
+const SocialIconContainer: FC<Props> = ({ position }) => {
+  const isBrowser = typeof window !== 'undefined'
+  const [screenWidth, setScreenWidth] = useState(
+    isBrowser ? window.innerWidth : 0,
+  )
+
+  useEffect(() => {
+    const handleResize = () => {
+      setScreenWidth(window?.innerWidth)
+    }
+
+    window.addEventListener('resize', handleResize)
+    return () => window.removeEventListener('resize', handleResize)
+  }, [])
+
+  if (screenWidth < getThemeLargeScreenWidth() && position === 'side') {
+    return null
+  }
+
+  return (
+    <div className={`flex flex-${position === 'side' ? 'col' : 'row'}`}>
+      {socialMediaDetails.map((detail) => (
+        <SocialMediaLink key={detail.href} {...detail} />
+      ))}
+    </div>
```

Comment:
**suggestion (non-blocking):** you could achieve the same thing using CSS and a little less JavaScript. The [`hidden` class](https://tailwindcss.com/docs/display#hidden) hides an element altogether, and [the `lg:flex` class](https://tailwindcss.com/docs/display#breakpoints-and-media-queries) will show the element at our large breakpoint.

```tsx
<div className={`flex flex-${position === 'side' ? 'col hidden lg:flex' : 'row'}`}>
  {socialMediaDetails.map((detail) => (
    <SocialMediaLink key={detail.href} {...detail} />
  ))}
 /div>
```

Reply:
Thank you for the review and feedback @deammer ! 

I thought about hiding the elements with CSS but then decided to use the JS approach instead because this way the elements aren't even rendered. With the CSS approach they would be rendered first and then hidden, wouldn't they? (I might be wrong, my frontend skills are a bit rusty!)

In my experience it also makes it easier to test (if we decided to start writing tests...) I remember having a hard time trying to assert that an element was hidden with CSS... It was easier to just return null instead.

Hope you don't mind if I leave it like this and merge!